examples/compose/plot_compare_reduction.py

```diff
@@ -53,35 +56,35 @@
         "classify__C": C_OPTIONS,
     },
     {
-        "reduce_dim": [SelectKBest(chi2)],
+        "reduce_dim": [SelectKBest(mutual_info_classif)],
         "reduce_dim__k": N_FEATURES_OPTIONS,
         "classify__C": C_OPTIONS,
     },
 ]
-reducer_labels = ["PCA", "NMF", "KBest(chi2)"]
+reducer_labels = ["PCA", "NMF", "KBest(mutual_info_classif)"]
 
 grid = GridSearchCV(pipe, n_jobs=1, param_grid=param_grid)
-X, y = load_digits(return_X_y=True)
 grid.fit(X, y)
```

Comment:
The `NMF` is raising a `ConvergenceWarning`. Maybe we can increase the number of iterations.

Reply:
scaling data is a much better way ;)

Reply:
We need both :)